tests/lib/edgehog_device/integration/prj.conf

```diff
@@ -13,18 +13,16 @@ CONFIG_TEST_LOGGING_DEFAULTS=y
 
 CONFIG_LOG=y
 
- # MbedTLS
+# MbedTLS
 CONFIG_MBEDTLS=y
 CONFIG_MBEDTLS_BUILTIN=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
-CONFIG_MBEDTLS_HEAP_SIZE=55000
+CONFIG_MBEDTLS_HEAP_SIZE=95000
```

Comment:
Is it right ti increment the HEAP_SIZE?

Reply:
It makes little difference in this case, as the integration tests don't use mbedtls. However, this larger value is the actual minimum heap size that an average Edgehog application requires.